src/app/shared/components/loader/loader.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { Subscription } from 'rxjs';
+import { LoaderService } from 'src/app/core/services/loader.service';
+import { debounceTime } from 'rxjs/operators';
+
+@Component({
+  selector: 'app-loader',
+  templateUrl: './loader.component.html',
+  styleUrls: ['./loader.component.scss']
```

Comment:
encapsulation: ViewEncapsulation.None

try to populate the component styles across all elements to touch-base the .spinner between
<app-root> tag